mathinterpreter/__main__.py

```diff
@@ -13,21 +13,28 @@ def main():
     while repl:
```

Comment:
I suggest to insert the test "len(sys.argv)==1" inside the cli function, and do something like this:

if len(sys.argv)>1:
    cli(sys.argv)
else:
    while True:
        ....

I believe that this way will improve readability.
Note that I do not like loops like "while True:", but it seems to be a common practice in python language, and it is compatible with the EAFP principle.